crates/bevy_app/src/schedule_runner.rs

```diff
@@ -121,7 +127,10 @@ impl Plugin for ScheduleRunnerPlugin {
                     {
                         loop {
                             match tick(&mut app, wait) {
-                                Ok(Some(delay)) => std::thread::sleep(delay),
+                                Ok(Some(_delay)) => {
+                                    #[cfg(feature = "std")]
+                                    std::thread::sleep(_delay);
```

Comment:
Without access to `std::thread::sleep`, `no_std` apps will run as-fast-as-possible by default. This isn't ideal, but I believe a reasonable compromise.